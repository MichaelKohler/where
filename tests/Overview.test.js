import React from 'react';
import { shallow } from 'enzyme';

import Overview from '../src/Overview';

test('should correctly shallow render Overview', () => {
  const component = shallow(<Overview />);
  expect(component).toMatchSnapshot();
});
