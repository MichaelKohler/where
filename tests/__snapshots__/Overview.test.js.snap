// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly shallow render Overview 1`] = `
<div
  className="container"
>
  <section>
    <div
      className="widget-row"
    >
      <NextTripWidget
        color="brand"
        nextTrip={
          Object {
            "country": "GR",
            "dateFrom": "2021-03-11",
            "dateUntil": "2021-03-28",
            "description": "",
            "destination": "Athens",
            "flights": 2,
            "lat": 38,
            "lng": 23.7,
          }
        }
      />
      <TotalWidget
        color="brand"
        totals={
          Object {
            "countries": 4,
            "trips": 5,
            "uniqueDestinations": 3,
          }
        }
      />
    </div>
    <Map
      selector="visitedMap"
      trips={
        Array [
          Object {
            "country": "CH",
            "dateFrom": "2020-07-30",
            "dateUntil": "2020-08-03",
            "description": "",
            "destination": "Bern",
            "flights": 2,
            "lat": 46.94,
            "lng": 7.43,
          },
          Object {
            "country": "GR",
            "dateFrom": "2020-07-03",
            "dateUntil": "2020-07-29",
            "description": "",
            "destination": "Athens",
            "flights": 2,
            "lat": 38,
            "lng": 23.7,
          },
          Object {
            "country": "CZ",
            "dateFrom": "2020-02-14",
            "dateUntil": "2020-02-16",
            "description": "",
            "destination": "Prague",
            "flights": 0,
            "lat": 50.08,
            "lng": 14.42,
          },
        ]
      }
    />
    <TripTable
      trips={
        Array [
          Object {
            "country": "GR",
            "dateFrom": "2021-03-11",
            "dateUntil": "2021-03-28",
            "description": "",
            "destination": "Athens",
            "flights": 2,
            "lat": 38,
            "lng": 23.7,
          },
          Object {
            "country": "CH",
            "dateFrom": "2020-07-30",
            "dateUntil": "2020-08-03",
            "description": "",
            "destination": "Bern",
            "flights": 2,
            "lat": 46.94,
            "lng": 7.43,
          },
          Object {
            "country": "GR",
            "dateFrom": "2020-07-03",
            "dateUntil": "2020-07-29",
            "description": "",
            "destination": "Athens",
            "flights": 2,
            "lat": 38,
            "lng": 23.7,
          },
          Object {
            "country": "CH",
            "dateFrom": "2020-02-18",
            "dateUntil": "2020-02-20",
            "description": "",
            "destination": "Bern",
            "flights": 2,
            "lat": 46.94,
            "lng": 7.43,
          },
          Object {
            "country": "CZ",
            "dateFrom": "2020-02-14",
            "dateUntil": "2020-02-16",
            "description": "",
            "destination": "Prague",
            "flights": 0,
            "lat": 50.08,
            "lng": 14.42,
          },
        ]
      }
    />
  </section>
</div>
`;
